compass/landice/tests/ismip6_run/ismip6_ais_proj2300/streams.landice.template

```diff
@@ -0,0 +1,159 @@
+
+<immutable_stream name="input"
+                  filename_template="{{ input_file_init_cond }}"
+                  input_interval="initial_only"
+                  type="input"/>
+
+<immutable_stream name="restart"
+                  filename_interval="output_interval"
+                  clobber_mode="overwrite"
+                  precision="double"
+                  output_interval="0001-00-00_00:00:00"
+                  reference_time="2000-01-01_00:00:00"
+                  filename_template="rst.$Y-$M-$D.nc"
+                  input_interval="initial_only"
+                  type="input;output"
+                  input_interal="initial_only"/>
+
+<stream name="regionsInput"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ input_file_region_mask }}"
+        input_interval="initial_only"
+        runtime_format="single_file">
+        <var name="regionCellMasks"/>
+</stream>
+
+<stream name="ismip6_smb"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ input_file_SMB_forcing }}"
+        input_interval="{{ forcing_interval }}"
+        reference_time="2000-01-01_00:00:00"
+        runtime_format="single_file">
+        <var name="sfcMassBal"/>
+</stream>
+
+<stream name="ismip6_TF"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ input_file_TF_forcing }}"
+        input_interval="{{ forcing_interval }}"
+        reference_time="2000-01-01_00:00:00"
+        runtime_format="single_file">
+        <var name="ismip6shelfMelt_3dThermalForcing"/>
+        <var name="ismip6shelfMelt_zOcean"/>
+</stream>
+
+<stream name="ismip6_params"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ input_file_melt_params }}"
+        input_interval="initial_only"
+        runtime_format="single_file">
+        <var name="ismip6shelfMelt_deltaT"/>
+        <var name="ismip6shelfMelt_basin"/>
+        <var name="ismip6shelfMelt_gamma0"/>
+</stream>
+
+<stream name="output"
+        clobber_mode="overwrite"
+        output_interval="0001-00-00_00:00:00"
+        reference_time="2000-01-01_00:00:00"
+        filename_interval="005-00-00_00:00:00"
+        filename_template="output/output_state_$Y.nc"
+        type="output">
+
+    <stream name="basicmesh"/>
+    <var name="xtime"/>
+    <var name="simulationStartTime"/>
+    <var name="thickness"/>
+    <var name="lowerSurface"/>
+    <var name="upperSurface"/>
+    <var name="bedTopography"/>
+    <var name="uReconstructX"/>
+    <var name="uReconstructY"/>
+    <var name="surfaceSpeed"/>
+    <var name="xvelmean"/>
+    <var name="yvelmean"/>
+    <var name="basalTemperature"/>
+    <var name="surfaceTemperature"/>
+    <var name="betaSolve"/>
+    <var name="daysSinceStart"/>
+    <var name="cellProcID"/>
+    <var name="cellMask"/>
+    <var name="deltat"/>
+    <var name="vonMisesStress"/>
+    <var name="damage"/>
+</stream>
+
+<stream name="output_flux"
```

Comment:
Add `faceMeltingThickness` and `faceMeltSpeed` here for 4km?

Reply:
These now get included when facemelting is activated in the cfg.